src/Components/ArtworkFilter/ArtworkFilterArtworkGrid.tsx

```diff
@@ -118,6 +130,9 @@ export const ArtworkFilterArtworkGridRefetchContainer = createFragmentContainer(
         }
         edges {
           node {
+            collectorSignals {
```

Comment:
I just remembered this!

As a heads up - cc @starsirius  - we typically don't want to include `collectorSignals` as a requested field in these fragments, vs. needing them to be deferred. This is because this field may return personalized information (like an 'offer' related signal), which will make caching grids for logged-in folks challenging/impossible.

I _think_ @starsirius has been working on moving these _out_ of the fragments - we should discuss where that's at before re-implementing.